vector/src/main/java/im/vector/app/features/login/LoginActivity.kt

```diff
@@ -252,7 +253,7 @@ open class LoginActivity : VectorBaseActivity<ActivityLoginBinding>(), UnlockedA
                 // It depends on the LoginMode
                 when (state.loginMode) {
                     LoginMode.Unknown,
-                    is LoginMode.Sso      -> error("Developer error")
+                    is LoginMode.Sso -> launchSsoFlow()
```

Comment:
Unknown and sso are doing the same thing?

Reply:
Unintentional. Good catch!

Reply:
yes, good catch @BillCarsonFr !